target/linux/ramips/dts/mt7620a_netgear_ex6100-v1.dts

```diff
@@ -0,0 +1,90 @@
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6100-v1", "ralink,mt7620a-soc";
+	model = "Netgear EX6100 v1";
+
+	aliases {
+		led-boot = &led_power_amber;
+		led-failsafe = &led_power_amber;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_amber;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_g {
+			label = "ex6100-v1:green:power";
+			gpios = <&gpio2 23 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led_power_amber: power_a {
+			label = "ex6100-v1:amber:power";
+			gpios = <&gpio2 28 GPIO_ACTIVE_LOW>;
+		};
+
+		router_g {
+			label = "ex6100-v1:green:router";
+			gpios = <&gpio2 25 GPIO_ACTIVE_LOW>;
+		};
+
+		router_r {
+			label = "ex6100-v1:red:router";
+			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
+		};
+
+		router_b {
+			label = "ex6100-v1:blue:router-arrow";
+			gpios = <&gpio2 29 GPIO_ACTIVE_LOW>;
+		};
+
+		device_g {
+			label = "ex6100-v1:green:device";
+			gpios = <&gpio2 20 GPIO_ACTIVE_LOW>;
+		};
+
+		device_r {
+			label = "ex6100-v1:red:device";
+			gpios = <&gpio2 21 GPIO_ACTIVE_LOW>;
+		};
+
+		device_b {
+			label = "ex6100-v1:blue:device-arrow";
+			gpios = <&gpio2 22 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "ex6100-v1:green:wps";
+			gpios = <&gpio2 27 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&ethernet {
+	pinctrl-names = "default";
+	pinctrl-0 = <&rgmii1_pins &mdio_pins>;
```

Comment:
I found there's `phy_reset_pins` in another device with RTL8211E:
https://github.com/openwrt/openwrt/blob/d2b8ccb1c04def81224da6f42f644c7d239b9986/target/linux/ramips/dts/mt7620a_edimax_ew-747x.dtsi#L123-L128 Now ethernet is working, is that necessary?
And it disabled phy0-4
https://github.com/openwrt/openwrt/blob/d2b8ccb1c04def81224da6f42f644c7d239b9986/target/linux/ramips/dts/mt7620a_edimax_ew-747x.dtsi#L139-L167 Should we need this too?

Reply:
So anybody know if it's necessary?

Reply:
Try to highlight the authors of the mentioned devices here ...

Reply:
It seems that the original author doesn't have a GitHub account.
https://github.com/openwrt/openwrt/commit/88f7a29f99fb8209f6d7100d0cc0f7af6dba89cf

Reply:
Ping @ynezz as he reviewed that patch.

Reply:
Write an e-mail to Birger. He typically responds relatively fast and is willing to help if he can.

Reply:
OK, sent.

Reply:
@adrianschmutzler Still no reply from Birger Koblitz. Is there anyone else know this?

Reply:
@ysc3839: Have you a reply from Birger Koblitz?
This GitHub account has been removed...

Reply:
@Neustradamus No.
By the way thanks for your remind, I should close this PR since I have no access to this device for a long time.